_core/smart/tutorial.md

```diff
@@ -1,211 +1,188 @@
+> To create private functions, you would use the `define-private` keyword. Private functions can only be executed by the current smart contract - not from other smart contracts published to the same network. Only public functions can be called from other contracts.
 
-    The function doesn't take any parameters and simply returns "hello world" using the [`ok`](https://docs.blockstack.org/core/smart/clarityref#ok) response constructor.
+The function doesn't take any parameters and simply returns "hello world" using the [`ok`](https://docs.blockstack.org/core/smart/clarityref#ok) response constructor.
 
-    Let's review the second public function, `echo-number`. As opposed to the function before, this takes an input parameter of the type [`int`](https://docs.blockstack.org/core/smart/clarityref#int-type). Along with integer, Clarity supports the following types:
-   * [uint](https://docs.blockstack.org/core/smart/clarityref#uint-type): 16-byte unsigned integer
-   * [principal](https://docs.blockstack.org/core/smart/clarityref#principal-type): spending entity, roughly equivalent to a Stacks address
-   * [boolean](https://docs.blockstack.org/core/smart/clarityref#bool-type): `true` or `false`
-   * [buffer](https://docs.blockstack.org/core/smart/clarityref#buffer-type): fixed-length byte buffers
-   * [tuple](https://docs.blockstack.org/core/smart/clarityref#tuple-type): named fields in keys and values
+Let's review the second public function, `echo-number`. As opposed to the function before, this takes an input parameter of the type [`int`](https://docs.blockstack.org/core/smart/clarityref#int-type). Along with integer, Clarity supports the following types:
+* [uint](https://docs.blockstack.org/core/smart/clarityref#uint-type): 16-byte unsigned integer
+* [principal](https://docs.blockstack.org/core/smart/clarityref#principal-type): spending entity, roughly equivalent to a Stacks address
+* [boolean](https://docs.blockstack.org/core/smart/clarityref#bool-type): `true` or `false`
+* [buffer](https://docs.blockstack.org/core/smart/clarityref#buffer-type): fixed-length byte buffers
+* [tuple](https://docs.blockstack.org/core/smart/clarityref#tuple-type): named fields in keys and values
 
-    The function simply uses the `ok` response and returns the value passed to the function.
+The function simply uses the `ok` response and returns the value passed to the function.
```

Comment:
Because this is "Get Started Writing Smart Contracts." After finishing it, they should have an environment that allows them to be productive. After running this tutorial the users will have exposure to tools that enable them to write and validate contracts with syntax highlighting and syntax error handling.

I think you're more highlighting how using pre-baked starter templates robs users of the experience of coding up a smart contract from scratch.

Reply:
I actually like the first paragraph - is this explanation clearly stated in the tutorial? With the expectation set, this is totally fine

Reply:
👍